man/os-release.xml

```diff
@@ -339,6 +339,13 @@
     name in order to avoid name clashes. Applications
     reading this file must ignore unknown fields. Example:
     <literal>DEBIAN_BTS="debbugs://bugs.debian.org/"</literal></para>
+
+    <para>Container runtime managers may make the host's identification
+    data available to applications by exposing each field as an environment
+    variable  as defined by <ulink
+    url="https://systemd.io/CONTAINER_INTERFACE/">the Container Interface
+    specification</ulink>. Example:
+    <literal>container_host_VERSION_ID=11.04</literal></para>
```

Comment:
I think we should map the four xyz_ID fields explicitly and use lowercase then, so that `$container_host_version_id=` is the name for the variable mentioned here. Let's not blanket import this env vars, but be specific to the four relevant ones.